src/imio/schedule/dashboard/columns.py

```diff
@@ -13,13 +13,34 @@
 from imio.schedule.dashboard.interfaces import ISimpleDisplayTaskStatus
 from imio.schedule.dashboard.interfaces import IStatusColumn
 from imio.schedule.interfaces import IToIcon
+from imio.schedule.utils import close_or_past_date
 
 from zope.component import queryAdapter
 from zope.component import queryMultiAdapter
 from zope.interface import implements
 from zope.i18n import translate
 
 
+def due_date_extra_classes(due_date, review_state):
+    extra_classes = ""
+
+    if review_state != 'closed' and due_date and due_date.year != 9999:
+        orange_limit = api.portal.get_registry_record(
+            'Products.urban.browser.schedule_settings.ISchedule.color_orange_x_days_before_due_date',
```

Comment:
Does not look right to have a reference to Products.urban when this package is a dependency of Products.urban.